src/app/components/pages/UserProfile.jsx

```diff
@@ -42,6 +42,16 @@ export default class UserProfile extends React.Component {
         this.loadMore = this.loadMore.bind(this);
     }
 
+    componentDidMount() {
+        window.addEventListener("resize", this.resize.bind(this));
+        this.resize();
+    }
+
+    resize() {
+        this.setState({showMobileCover: window.innerWidth <= 639});
+        this.forceUpdate();
+    }
+
```

Comment:
@netuoso thanks for this. I would prefer to not have to manage the responsive behavior in state like this. We have foundation styles available and this behavior could be  implemented by applying the classname 'show-for-small' to the new mobile image and 'hide-for-small' to the existing one.
https://foundation.zurb.com/sites/docs/v/3.2.5/media-queries.php#screenCode

Example:
https://github.com/steemit/condenser/blob/master/src/app/components/modules/TopRightMenu.jsx#L53